Shared/CloudinaryVersion.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace CloudinaryDotNet
+{
+    /// <summary>
+    /// Contains information about release version.
+    /// </summary>
+    internal static class CloudinaryVersion
+    {
+        /// <summary>
+        /// Get full version number of .NET SDK.
+        /// </summary>
+        public static string Full => "1.11.0";
```

Comment:
What supposed to update this when the publish scripts are running?

Reply:
Although from @const-cloudinary 's past comments we understood that manual version change is acceptable, it makes sense to automate this step during publish. We will do it in the current sprint.

Reply:
If @const-cloudinary is OK with it as well.

Reply:
@asisayag2 , @RTLcoil , there is a set_version.ps1 file that updates the version, I updated it.

Reply:
@RTLcoil and version should be current version, when new version is released, a separate commit will be made.

Reply:
Cool, @const-cloudinary . So you'll let us know or would you like us to ping you on this at some point?

Reply:
@RTLcoil it all done, nothing to do here